HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -1357,9 +1357,16 @@ def from_oga(software_info)
   end
 
   class VacancyPeriod < BaseElement
-    ATTRS = [:begin_month, :begin_day, :end_month, :end_day]
+    ATTRS = [:begin_month, :begin_day, :begin_hour, :end_month, :end_day, :end_hour, :natvent]
```

Comment:
Why did you add natvent to this? I thought it only applies to power outages (hence why there's only a single new `schedules_power_outage_window_natvent_availability` argument in the BuildResHPXML measure).

Reply:
I see. So I guess you're adding it as an attribute here so that this class is 100% consistent with power outages and you can loop through both objects.

I guess I was mostly confused by the fact that it shows up as an input in EPvalidator.xml. Do we really want it as a user input?

Reply:
Removed from EPvalidator.xml.